tox.ini

```diff
@@ -12,4 +13,13 @@ deps =
 
 commands =
     pylama mozdownload
-    py.test {posargs}
+    coverage run --source mozdownload -m py.test {posargs}
+
+[testenv:clean]
+commands=
+  coverage erase
+
+[testenv:stats]
+commands=
+    coverage report --omit='.tox/*'
+    coverage html --omit='.tox/*'
```

Comment:
So the html report will contain more details? Especially so we can see which lines are not covered yet in each of the existing files?